geoengine/workflow_builder/blueprints.py

```diff
@@ -1,18 +1,11 @@
 '''This module contains blueprints for workflows.'''
 
 from . import operators
-from .. import api
 
 
 def sentinel2_band(band_name, provider="5779494c-f3a2-48b3-8a2d-5fbba8c5b6c5", utm_zone="UTM32N"):
     '''Creates a workflow for a band from Sentinel 2 data.'''
-    band_source = operators.GdalSource(
-        dataset=api.ExternalDataId(
-            type="external",
-            providerId=provider,
-            layerId=f"{utm_zone}:{band_name}"
-        )
-    )
+    band_source = operators.GdalSource(f"_:{provider}:{utm_zone}:{band_name}")
```

Comment:
I think this does not work. I get the following error when I try to register this workflow:

> 2023-06-06T12:16:08.698151Z  WARN Request{request_id=cfc0003f-7652-4a1d-90e0-62fcf78df514}: tracing_actix_web::middleware: Error encountered while processing the incoming HTTP request: ErrorResponse { error: "BodyDeserializeError", message: "named data must consist of at most three parts at line 1 column 127" }

Reply:
[444fd3f](https://github.com/geo-engine/geoengine-python/pull/136/commits/444fd3f24b6f2833b8d4df03cbb2d7c66add5da8)

Reply:
For the providers, you can put stuff in backticks. Then, you can put arbitrary stuff in there, which will also help for the ebv stuff with JSON.
It does not change the limitation for dataset names, just for NamedData.